### ISSUE TITLE
eslint --rule can't handle shell line breaks

### ISSUE THREAD

**Issue by minj** [eslint/eslint#9880]
**Tell us about your environment**

* **ESLint Version:** v4.15.0
* **Node Version:** v9.4.0
* **npm Version:** 5.6.0

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

See #9879

```bash
eslint --no-eslintrc --rule 'padding-line-between-statements: \
                              [error, \
                                      {blankLine: always, prev: *, next: debugger} \
                              ]' test5.js
```

**What did you expect to happen?**
It should just work™
**What actually happened? Please include the actual, raw output from ESLint.**
```
CLI:
        Configuration for rule "padding-line-between-statements" is invalid:
        Severity should be one of the following: 0 = off, 1 = warn, 2 = error (you passed '"\\[error"').

Error: CLI:
        Configuration for rule "padding-line-between-statements" is invalid:
        Severity should be one of the following: 0 = off, 1 = warn, 2 = error (you passed '"\\[error"').

    at Object.validateRuleOptions (/usr/lib/node_modules/eslint/lib/config/config-validator.js:115:15)
    at CLIEngine.Object.keys.forEach.name (/usr/lib/node_modules/eslint/lib/cli-engine.js:416:27)
    at Array.forEach (<anonymous>)
    at new CLIEngine (/usr/lib/node_modules/eslint/lib/cli-engine.js:415:45)
    at Object.execute (/usr/lib/node_modules/eslint/lib/cli.js:187:28)
    at Object.<anonymous> (/usr/lib/node_modules/eslint/bin/eslint.js:74:28)
    at Module._compile (module.js:660:30)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
```

**Comment by not-an-aardvark** [comment]
I don't think backslash-escaping works in general when using single-quoted strings. (This is a bash behavior, before ESLint gets run.) As a result, the input that ESLint receives contains backslashes, which it doesn't know how to interpret.

Instead, you should use double-quoted strings (if you don't want to have linebreaks in the input), or simply omit the backslashes if it's okay to have linebreaks in the input.

**Comment by minj** [comment]
Right, silly me.
